variables.tf

```diff
@@ -4,6 +4,12 @@ variable "api_url" {
   default     = "https://api.cast.ai"
 }
 
+variable "castai_api_token" {
+  type = string
+  description = "CAST AI API token created in console.cast.ai API Access keys section."
+  sensitive = true
```

Comment:
Shouldn't this be also optional since waiting is optional?

Reply:
ATM the logic is that wait is optional but by default is enabled. Soo this somehow makes it required. Hmm maybe we should make wait optional but by default disabled and then this will be optional?

Reply:
I would say `everything new is optional` otherwise you break customers plans when they upgrade

Reply:
Done, now all fields are optional.